pkg/utils/restic/pod.go

```diff
@@ -275,42 +278,31 @@ func PodSpecRestore(restore *extensionv1.Restore, resticId string, params PodSpe
 				},
 			),
 		},
-		VolumeMounts: volumeMounts,
+		VolumeMounts: resticRestoreVolumeMounts,
 	})
 
-	for i, _ := range containers {
-		containers[i] = WrapContainer(containers[i], siteId, restore.ObjectMeta.Namespace)
-	}
-	for i, _ := range initContainers {
-		initContainers[i] = WrapContainer(initContainers[i], siteId, restore.ObjectMeta.Namespace)
-	}
-
-	spec := corev1.PodSpec{
-		RestartPolicy:  corev1.RestartPolicyNever,
-		InitContainers: initContainers,
-		Containers:     containers,
-		Volumes: AttachVolume([]corev1.Volume{
-			{
-				Name: VolumeMySQL,
-				VolumeSource: corev1.VolumeSource{
-					EmptyDir: &corev1.EmptyDirVolumeSource{
-						Medium: corev1.StorageMediumDefault,
-					},
+	// Volume definitions for the pod.
+	specVolumes := AttachVolume([]corev1.Volume{
+		{
+			Name: VolumeMySQL,
+			VolumeSource: corev1.VolumeSource{
+				EmptyDir: &corev1.EmptyDirVolumeSource{
+					Medium: corev1.StorageMediumDefault,
```

Comment:
This `Medium` can be omitted as it is the default.

Reply:
Donezo!